notebooks_tsqr/NightLog.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    date = ut.dos2dt(day_obs)
</pre><p>day obs string to datetime - get_datetime_from_day_obs_str(), if and when we need a _int() version, then just the one function should cover the day_obs in any form parameter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>